### PR TITLE
Add player component and model

### PR DESCRIPTION
**Pull request by Hans-Zhang28**

### CODE REVIEW

File: src/components/:x [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```
Comment (19506jk) [inline src/components/:x]: What is this file?

File: src/models/playerModel.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export default class PlayerModel {
+  constructor(name, isLead) {
+    console.log(name);
```
Comment (19506jk) [inline src/models/playerModel.js]: To be removed

Comment (Hans-Zhang28) [reply src/models/playerModel.js]: Done

File: src/components/:x (hunk 1 above)
Comment (Hans-Zhang28) [reply src/components/:x]: Removed

**Pull request merged**